### PR TITLE
21373 csv modifier sample broken

### PR DESCRIPTION
**Pull request by jomar-honsi**
Replaced `InvertModifier `with `beforeParse`. Removed `ChainModifier`.

### CODE REVIEW

**Review by bre1470 — APPROVED** [review]
Nice! Works for me. 👍

File: docs/dashboards/data-modifiers.md [hunk 1]
```diff
@@ -98,3 +98,6 @@ You can check the modifiers in action with the following demos:
 
 #### DataGrid with MathModifier
 <iframe style="width: 100%; height: 700px; border: none;" src="https://www.highcharts.com/samples/embed/dashboards/demo/datagrid-mathmodifier" allow="fullscreen"></iframe>
+
+#### Using CSV data and RangeModifier
+<iframe style="width: 100%; height: 733px; border: none;" src="https://www.highcharts.com/samples/embed/highcharts/data-tools/csv-modifiers" allow="fullscreen"></iframe>
```
Comment (bre1470) [inline docs/dashboards/data-modifiers.md:103]: One thing though. Data tools is only internal to test the data layer implementation. So if you like to link it, you should move the demo into some other demo and without data tools bundle.

**Review by karolkolodziej — APPROVED** [review]
Do you also get that: 
image
The legend doesn't control the charts?

**Comment by bre1470** [comment]
Yes, nothing in sync for me. Should it be in sync? Also I do not know the recent design goals when it comes to the Dashboards API. Back in my days the focus was on options-driven solutions.

**Comment by jomar-honsi** [comment]
It is possible that the original had sync enabled, I will check and if possible reintroduce it.

**Comment by jomar-honsi** [comment]
> Yes, nothing in sync for me. Should it be in sync? Also I do not know the recent design goals when it comes to the Dashboards API. Back in my days the focus was on options-driven solutions.

Fixed.

**Comment by sebastianbochan** [comment]
So we can merge it :)

**Review by sebastianbochan — APPROVED** [review]

File: docs/dashboards/data-modifiers.md (hunk 1 above)
Comment (sebastianbochan) [reply docs/dashboards/data-modifiers.md:103]: You can move it to the `dashboards/data/` (create a folder if needed).

**Pull request merged**